mentat/sampler/README.md

```diff
@@ -0,0 +1,23 @@
+# Mentat Sampler API
+The Sampler API is an open-source standard to captures interactions between a developer and an LLM-based AI Assistant. It's intended to facilitate sharing of benchmarks and fine-tuning data throughout the open-source community and industry.
+
+| Field            | Type          | Description |
+|------------------|---------------|-------------|
+| title            | `str`         | plaintext by creator |
+| description      | `str`         | plaintext by creator |
+| id               | `uuid`        |  |
+| parent_id        | `uuid`        | id of sample immediately before this |
```

Comment:
`parent_id` is set if you make multiple samples in the same conversation? What's the point of tracking that information?

Reply:
I'm imagining if you have 3 interactions in a row, you could use it to benchmark or file-tune an agent LLM.